app/views/tips/show.html.erb

```diff
@@ -5,4 +5,6 @@
   <p><%= @tip.body %></p>
   <hr>
   <button type="button" class="btn btn-info"><%= link_to "Edit", edit_tip_path(@tip) %></button>
+  <%= link_to "Delete tip", delete_tip_path(@tip), :method => 'get', :data => {:confirm => 'Are you sure you want to delete this post?'} %>
```

Comment:
use `tip_path(@tip)`. Remove `delete_tip` alias from your routes. The path is the same for both show and delete, so there's no need for a second alias.